indico/modules/events/abstracts/util.py

```diff
@@ -197,15 +199,25 @@ def get_roles_for_event(event):
     :return: A dictionary in the form ``{track: {role: [users]}}``
     """
     roles = defaultdict(dict)
-    for track in Track.query.with_parent(event).options(joinedload('conveners'), joinedload('abstract_reviewers')):
+    roles['*'].setdefault('reviewer', [])
+    roles['*'].setdefault('convener', [])
+    for track in Track.query.with_parent(event).options(subqueryload('acl_entries')):
         roles[str(track.id)].setdefault('reviewer', [])
         roles[str(track.id)].setdefault('convener', [])
-        for reviewer in track.abstract_reviewers:
-            roles[str(track.id)]['reviewer'].append(reviewer.id)
-        for convener in track.conveners:
-            roles[str(track.id)]['convener'].append(convener.id)
-    roles['*']['reviewer'] = [reviewer.id for reviewer in event.global_abstract_reviewers]
-    roles['*']['convener'] = [reviewer.id for reviewer in event.global_conveners]
+        for principal in track.acl_entries:
+            if not principal.user_id:
```

Comment:
check `principal.type` instead. user 0 may actually be a real user for anyone who migrated from 1.2